releasenotes/notes/fill=dropna-9e055895e7bff778.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Aggregates API and cross metrics aggregation API can take dropna for
+    fill. This acts like 'null', but NAN values are removed from the result.
```

Comment:
"for the `fill` parameter."

NAN or NaN?

Reply:
NaN